src/ride/ride.h

```diff
@@ -201,18 +201,9 @@ typedef struct {
 	uint8 pad_11C[0x2];
 	uint8 num_sheltered_sections;   // 0x11E
 	uint8 var_11F;
-	sint16 var_120;
-	sint16 var_122;
-	sint16 var_124;
-	sint16 var_126;
-	sint16 var_128;
-	sint16 var_12A;
-	sint16 var_12C;
-	sint16 var_12E;
-	uint16 age;						// 0x130
-	sint16 running_cost;			// 0x132
-	sint16 var_134;
-	sint16 var_136;
+	uint16 customer_counter;        // Customer count in the current 960 game tick (about 30 seconds) interval
```

Comment:
make sure that the offset comments are still present. Move your extra comment above the field.